harness/o.n.insane/src/org/netbeans/insane/model/InsaneConverter.java

```diff
@@ -351,8 +351,8 @@ private void compute() {
         objsOffset = currentOffset;
         
         // compute offsets of instances
-        for (Iterator<InstanceInfo> it = instanceInfo.iterator(); it.hasNext(); ) {
-            InstanceInfo info = it.next();
+        for (Iterator it = instanceInfo.iterator(); it.hasNext(); ) {
+            InstanceInfo info = (InstanceInfo)it.next();
```

Comment:
it fixes compilation error

Reply:
The compilation error is caused by using the wrong generics in `ObjectSet`, you reverted the introduced generic. Exactly these kind of problems are the reason to be very careful to quickly introduce generics, there is a high chance, that it will be not the right type.